recipes/make-sense/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "make-sense" %}
+{% set version = "1.6.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SkalskiP/make-sense/archive/{{ version }}-alpha.tar.gz
+  sha256: c61b2a5926c93da57694a42f29efd14cb99f1fb313be76269babe6efb78bbecb
+
+build:
+  number: 0
+  script:
+    - tgz=$(npm pack)  # [not win]
+    - npm install -g $tgz  # [not win]
+    - FOR /F "delims=" %%i IN ('npm pack') DO set "tgz=%%i"  # [win]
+    - npm install -g %tgz%  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```

Comment:
Do you need a C++ compiler for an npm package?

Reply:
yes, i tried building without and kept getting an error that required the compiler

Reply:
Weird.... CC @isuruf @beckermr

Reply:
It's compiling libsass which isn't installed itself but is used for generating the installed CSS files